kai/rpc_server/main.py

```diff
@@ -66,10 +66,9 @@ def main() -> None:
     # simply IO[bytes], rather than as BufferedReader and BufferedWriter for
     # some reason, at least on my machine.
     rpc_server = JsonRpcServer(
-        json_rpc_stream=BareJsonStream(
+        json_rpc_stream=LspStyleStream(
```

Comment:
do we still want to remove log?

Reply:
The LspStyleStream doesn't have an `__init__` that accepts the log param. So I just dropped it.